crate_universe/private/crate.bzl

```diff
@@ -175,37 +175,42 @@ def _annotation(
         for patch in patches:
             _assert_absolute(patch)
 
+    def stringify_list(values):
```

Comment:
Can you comment why this is needed? Also can it be private?

Reply:
And can we not nest the definition of stringify_list inside this function?